dashboard/db/migrate/20210722193724_setup_contained_levels.rb

```diff
@@ -0,0 +1,5 @@
+class SetupContainedLevels < ActiveRecord::Migration[5.2]
+  def change
+    Level.all.each(&:setup_contained_levels)
```

Comment:
this takes about two minutes to run locally, so it's not absurd but also not ideal. Open to suggestions on a better way to do a one-off migration of existing levels.

Reply:
I agree that 2 minutes seems okay. It is possible that more levels exist on production than any other environments but I doubt many of those have contained levels.

Reply:
FWIW, this took 3 minutes on my machine (182 seconds according to the rails output). Still seems pretty reasonable, but there might be a bit of variation